ProcessLib/LIE/Common/FractureProperty.h

```diff
@@ -54,11 +60,43 @@ inline void setFractureProperty(unsigned dim, MeshLib::Element const& e,
     // 1st node is used but using other node is also possible, because
     // a fracture is not curving
     for (int j = 0; j < 3; j++)
-        frac_prop.point_on_fracture[j] = e.getNode(0)->getCoords()[j];
+        frac_prop.point_on_fracture[j] = e.getCenterOfGravity().getCoords()[j];
     computeNormalVector(e, dim, frac_prop.normal_vector);
     frac_prop.R.resize(dim, dim);
     computeRotationMatrix(e, frac_prop.normal_vector, dim, frac_prop.R);
 }
 
+inline BranchProperty* createBranchProperty(MeshLib::Node const& branchNode,
```

Comment:
I see that this function is called to fill ``_process_data._vec_junction_property`` which is a vector of unique_ptr. I think that you can return `std::unique_ptr<BranchProperty>` in this function.  Then use std::move in line 139 of file
ProcessLib/LIE/SmallDeformation/SmallDeformationProcess.cpp.

Reply:
It is just related to the operator of `new`. If a `new` is used, one may try to find a `delete`. If you prefer to use it, it is OK. All other stuffs are OK for me too.

Reply:
benchmarks failed.